README.md

```diff
@@ -3,25 +3,26 @@
 Patchmanager is a tool that can be used to modify the Sailfish OS user experience.
 It is based on AUSMT (Auto-Update System Modification Technology), a set of scripts that enables system file patching.
 
-Patchmanager does not have application icon inside launcher. After installation Patchmanager can be found inside Settings.
+Patchmanager does not have application icon on the launcher.  After installation Patchmanager can be found inside Settings.
 
 ## For users
 
-To install Patchmanager you first need to install Warehouse https://openrepos.net/content/basil/warehouse-sailfishos, then install Patchmanager 2 using Warehouse.
+To install Patchmanager you first need to install Storeman (https://openrepos.net/content/osetr/storeman), then install Patchmanager 3 from OpenRepos using Storeman. 
```

Comment:
Storeman link seems to not be correct anymore - https://openrepos.net/content/osetr/storeman-installer works though

Reply:
Yes, well that happens if MRs are stalled for long: Information becomes outdated.
Thanks for reviewing and noticing.
Fixed now.